charts/astronomer/templates/houston/helm-hooks/houston-au-strategy-job.yaml

```
@@ -52,6 +52,8 @@ spec:
           # If you supply only args for a Container, the default Entrypoint defined in the Docker image is run with the args that you supplied.
           args: ["update-deployments-resource-mode"]
           securityContext: {{ toYaml .Values.securityContext | nindent 12 }}
+          resources:
+{{ toYaml .Values.houston.resources | indent 12 }}
```

`nindent` adds `\n` at the beginning, and using it makes the template a bit cleaner.
```suggestion
          resources: {{ toYaml .Values.houston.resources | nindent 12 }}
```